test/integration/rt_channel_test.exs

```diff
@@ -2332,6 +2346,194 @@ defmodule Realtime.Integration.RtChannelTest do
     end
   end
 
+  describe "measure traffic" do
+    setup %{tenant: tenant} do
+      events = [
+        [:realtime, :channel, :output_bytes],
+        [:realtime, :channel, :input_bytes]
+      ]
+
+      name = :"TestCounter_#{tenant.external_id}"
+
+      {:ok, _} =
+        start_supervised(%{
+          id: 1,
+          start: {Agent, :start_link, [fn -> %{} end, [name: name]]}
+        })
+
+      RateCounterHelper.stop(tenant.external_id)
+      on_exit(fn -> :telemetry.detach({__MODULE__, tenant.external_id}) end)
+      :telemetry.attach_many({__MODULE__, tenant.external_id}, events, &__MODULE__.handle_telemetry/4, name)
+
+      measure_traffic_interval_in_ms = Application.get_env(:realtime, :measure_traffic_interval_in_ms)
+      Application.put_env(:realtime, :measure_traffic_interval_in_ms, 10)
+      on_exit(fn -> Application.put_env(:realtime, :measure_traffic_interval_in_ms, measure_traffic_interval_in_ms) end)
```

Comment:
BTW this test file is async: true so changing env would probably move this to be async: false :sadpanda:

Maybe we just create a new integration file just for measure traffic and make it async false?

that's what I did for the distributed integration test that uses `Clustered`: https://github.com/supabase/realtime/blob/main/test/integration/distributed_realtime_channel_test.exs